### ISSUE TITLE
Using Gumbel-softmax to solve more complex environment

### ISSUE THREAD

**Issue by yunanyan** [shariqiqbal2810/maddpg-pytorch#42]
Have you tried to use Gumbel-Softmax to solve discrete action environments other than the particle environment? I tried to apply it to Starcraft II but its performance is really bad.
